tests/containers/forms/LogActivityForm.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import LogActivityForm from '../../../src/containers/forms/LogActivityForm';
+
+describe('<Stats />', () => {
+  it('should render withour crashing', () => {
+    expect(mount.bind(null, <LogActivityForm />)).not.toThrow();
+  });
+  it('should show the <DateField/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
```

Comment:
This and the next line have been repeated in every it block. They can be included in the `beforeEach` setup function